repose-aggregator/components/filters/simple-rbac-filter/src/main/scala/org/openrepose/filters/simplerbac/SimpleRbacFilter.scala

```diff
@@ -263,10 +285,11 @@ class SimpleRbacFilter @Inject()(configurationService: ConfigurationService,
              |$params$methods
              |    </resource>""".stripMargin
         }.mkString("\n")
-        val footer = s"""  </resources>
+        val footer = s"""
+                        |  </resources>
                         |</application>""".stripMargin
 
-        Some(s"$header\n$resources\n$footer")
+        Some(s"$header$resources$footer")
     }
   }
 
```

Comment:
I thought we were just going to add this once at the top, it looks like you are adding it basically everywhere?

Reply:
You have to add it everywhere because of WADL.

Reply:
to be clear i was talking about the new param, if it is needed everywhere...le sigh

Reply:
Yep. And, yep.

Reply:
The issue it that a `param` can be a direct sub of a `resource`, but a `request` can't be. A `request` must be a sub of a `method`.